python/ray/tests/test_autoscaler.py

```diff
@@ -3561,6 +3561,17 @@ def __init__(self, *args, **kwargs):
                     monitor.run()
                 mock_publish.assert_called_once()
 
+    def testAutoscalerResourceValidation(self):
+        from ray.autoscaler.sdk import request_resources
+
+        self.assertRaises(TypeError, request_resources(bundles={"CPU": 1}))
+        self.assertRaises(TypeError, request_resources(bundles=[{"GPU": 1}]))
```

Comment:
@DmitriGekhtman Is `[{"GPU": 1}]` a valid bundle value or we always would require "CPU" to be present ?

Reply:
`[{"GPU": 1}]` is valid.